### ISSUE TITLE
Missing function `format_sumstats_for_magma_crossplatform()`

### ISSUE THREAD

**Issue by bschilder** [neurogenomics/MAGMA_Celltyping#27]
Hi there, 

In the tutorial if references a user-provided function, `format_sumstats_for_magma_crossplatform()` but i can't seem to find this in the package.

Also, when I run `format_sumstats_for_magma()` it can't find the SNP column in the example (called "variant"). I think this col name is common enough it might be worth adding to the list of possible names for the SNP col.

Thanks, 
Brian

**Comment by NathanSkene** [comment]
Hey,

I got rid of `format_sumstats_for_magma_crossplatform` as a seperate function on one of the recent commits, figured I should make it the main function. Thanks for flagging this up.

The GWAS file that was used as an example got dropped from the dropbox location. Last time I checked it wasn't possible to download it anymore. Have been wanting to figure out an alternative but haven't had a chance to do so. Are you actually able to download something at the moment then from that link?

**Comment by NathanSkene** [comment]
Have been kinda meaning to split the `format_sumstats_for_magma` function off as a seperate package and set it up with proper example files, and check/tests to ensure it still works as expected. It's a bit of a pain of a function. Don't understand why the community can't agree on a set of columns that should be in a sumstats file.

**Comment by bschilder** [comment]
Got it, that makes sense!

As for the example sum stats file, I am still able to download it from the DropBox link. Though I noticed on the [Neale Lab's website](http://www.nealelab.is/uk-biobank) that there was a note about removing some of the older GWAS back in 2019 (though maybe they decided to keep this one?)

I couldn't agree more about GWAS sum stats standards... I attempted to tackle this issue within my own package, [echolocatoR](https://github.com/RajLabMSSM/echolocatoR). My solution was to just have the user specify the column name mappings in the function (and then calculate some other cols), but perhaps some mixture of these approaches would be useful. Having it within R is useful too. Along those lines, there's a `munge` function within [GenomicSEM](https://rdrr.io/github/MichelNivard/GenomicSEM/man/) (as well as an R wrapper for LDSC).
